client/js/components/faq/DpFaqSupportCard.vue

```diff
@@ -0,0 +1,109 @@
+<license>
+(c) 2010-present DEMOS plan GmbH.
+
+This file is part of the package demosplan,
+for more information see the license file.
+
+All rights reserved
+</license>
+
+<template>
+  <div>
+    <section v-if="title">
+      <h4 class="font-semibold">
+        {{ title }}
+      </h4>
```

Comment:
shouldn't h4 bring the correct style out of the box? Is semi-bold necessary?
@spiess-demos Do You have a opinion about that?

Reply:
when i remove it here, the boldness is not enough